backend/src/server.ts

```diff
@@ -19,7 +19,7 @@ const app = fastify({
 
 app.register(initializeApp);
 
-app.listen(PORT, IP, (err) => {
+app.listen({ port: Number(PORT), host: IP}, (err) => {
```

Comment:
Weird to translate to Number here... It's an env var, we should purify it at source.

Reply:
I agree, try to process the `PORT` here https://github.com/opendatahub-io/odh-dashboard/blob/main/backend/src/utils/constants.ts#L5

Reply:
Ok, yup, let me do that in the env var!

Reply:
resolved